src/components/NameForm.tsx

```diff
@@ -20,15 +58,16 @@ class NameForm extends React.Component<Props, State> {
     render() {
         return (
             <div>
-                <label>
+                <Label>
                     Podaj kwotę:
-                </label>
-                <input
+                </Label>
+                <Input
                     className="input"
                     type="number"
                     value={this.state.value}
                     onChange={this.handleChange}/>
-                <button className="button" onClick={this.handleSubmit}>Zatwierdź</button>
+                <Button onClick={this.handleSubmit}>Zatwierdź</Button>
+                <Rotate>Hedgehog</Rotate>
```

Comment:
I will for now approve that XDD

Reply:
I removed it from render, but we still can use it later for some, hmm,  reasons